util/Utils.re

```diff
@@ -250,3 +250,22 @@ let getEnvVar = (~env=Unix.environment()->Array.to_list, varToFind) => {
     | _ => None
   }
 }
+
+/*
+ * Quotes filename when not quoted
+ * Example:
+ * myFile.exe -> 'myFile.exe'
+ * 'myFile.exe' -> 'myFile.exe'
+ */
+let maybeQuoteFilename = (filename) => {
+  if (String.length(filename) < 1) {
```

Comment:
can't you just do `filename === ""`?

Reply:
`s/===/==/`

Reply:
ooops, indeed

Reply:
Should I compare the string? Or is the length ok ?

Reply:
just a nit. whatever you want. I would also change the length to a variable given you're using it later.

Reply:
Stored the length in a variable and force pushed.

Reply:
Thanks!